<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmShutdownBoxUi</class>
 <widget class="QWidget" name="ConfirmShutdownBoxUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>305</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>632</width>
    <height>305</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>632</width>
    <height>305</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_confirmExit">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Confirm Shutdown</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_shutdownComputer">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>270</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../playsurface-icons.qrc">
         <normaloff>:/QResources/icons/Shutdown-Computer.png</normaloff>:/QResources/icons/Shutdown-Computer.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>260</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_shutdownPlaysurfaceLauncher">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>270</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../playsurface-icons.qrc">
         <normaloff>:/QResources/icons/Shutdown-PlaysurfaceLauncher.png</normaloff>:/QResources/icons/Shutdown-PlaysurfaceLauncher.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>260</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>270</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../playsurface-icons.qrc">
         <normaloff>:/QResources/icons/Cancel-Return-PlaysurfaceLauncher.png</normaloff>:/QResources/icons/Cancel-Return-PlaysurfaceLauncher.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>260</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../playsurface-icons.qrc"/>
 </resources>
 <connections/>
</ui>
